public/posts/v1.7.md

```diff
@@ -0,0 +1,212 @@
+Today we are releasing Deno 1.7.0. This release contains many new features, some
+stabilizations, a some great improvements to existing APIs and tooling.
```

Comment:
```suggestion
stabilizations, and some great improvements to existing APIs and tooling.
```

Reply:
to piggy back on this comment, the comma before "and" should also be removed

Reply:
>  the comma before "and" should also be removed

It is an oxford comma. Our previous blog posts generally have these. (although sometimes I _am_ a little liberal with my use of commas 😆 ) https://en.wikipedia.org/wiki/Serial_comma